src/screens/CreateDetails/index.js

```diff
@@ -114,68 +99,46 @@ const CreateDetails = () => {
     if (collectionData.length > 0) {
       var array = [];
       collectionData.map(item => {
-        array.push({ id: item.collectionId, name: item.title });
+        array.push({ id: item.address, name: item.title });
       });
       setCollections(array);
     }
   }, [collectionData]);
 
   useEffect(() => {
-    let col_id = collectionData.filter(item => item.title === property)[0]?.collectionId;
-    setCollectionId(col_id);
+    let col_address = collectionData.filter(item => item.title === property)[0]?.address;
```

Comment:
Not sure if question mark `?` works here. It is not typescript and as far as I know, that works only in typescript.